vci-issuers.json

```diff
@@ -883,6 +883,7 @@
     {
       "iss": "https://interconnect.wellspan.org/interconnect-prd-fhir/api/epic/2021/Security/Open/EcKeys/32001/SHC",
       "name": "WellSpan Health"
+      "website": "https://www.wellspan.org/"
```

Comment:
not sure if this PR is still relevant, but you are missing a `,` on the line above, hence why the PR checks are failing (ditto on below changes to website)

Reply:
Yikes! Thanks for pointing that out! Unfortunately it doesn't look like that alone was enough to resolve the checks.

Reply:
Looks like these iss values don't/no longer resolve to a valid JSON document constituting the signing key of an issuer. For instance, viewing https://interconnect.wellspan.org/interconnect-prd-fhir/api/epic/2021/Security/Open/EcKeys/32001/SHC/.well-known/jwks.json returns an XML document, which does not conform to the SHC spec. I would investigate from there!

Reply:
Sorry, I think my browser was just attempting to render as XML; `curl`-ing does should that it is in fact returning a JSON document, so don't believe that's the issue.